test/mlir/onnx/onnx_lowering.mlir

```diff
@@ -2132,3 +2132,67 @@ func @cast_lowering_f64f32_10(%arg0: tensor<10xf64>) -> tensor<*xf32> {
   // CHECK: affine.store [[FPTRUNC]], [[RES]][%arg1] : memref<10xf32>
   // CHECK: return [[RES]] : memref<10xf32>
 }
+
+// -----
+
```

Comment:
Please add a comment to state what the tests are doing. I know we have not done it much in the past, they require it in MLIR and I think its a good thing. Please do for both,

Reply:
Thanks! I added them.